### ISSUE TITLE
GGMAX Enhancement - STOPWATCH behaviour

### ISSUE THREAD

**Issue by firstbastion** [TheGameCreators/GameGuruRepo#5831]
Requesting a new stopwatch TIMER behaviour

It is a timer that trigger/starts at zero 00:00:00 and counts upward as a race or event is going on;  for example when it is triggered at the start of the race or event, displays the active timer on screen, x=50 y=10 (can be changed)   and stops the count at the finishline trigger zone,  and then continues to display that elapsed time for another 5-10 second as it fades away with the option to trigger a completion sound.   

And hopefully displaying  atleast minutes:seconds:milliseconds - 00:00:00  (though the milliseconds could be omitted.)

**Comment by Necrym59** [comment]
Update your workshop there is one called Stop Watch that should do as you need

**Comment by firstbastion** [comment]
Thank you Necrym

**Comment by firstbastion** [comment]
There may still be a little finetuning needed.  It mostly works. 

I tried a standalone test of the stop watch  and it functioned as expected in the first minute, at 00:59: the minute column changed from 0 to 1,  

I let the timer continue running and at 1:59,  the minute column did not change from 1 to 2,  and the seconds count continued into the 60-70-80 seconds range, 

Continuing to let it run for 3 minutes  I took this snapshot to show the current functioning.  

![1-stopwatch-test29_1](https://github.com/user-attachments/assets/ff567bbf-ea40-4a09-b566-4cbba1ab4a3f)
